.github/workflows/master_build_merge.yml

```diff
@@ -0,0 +1,184 @@
+        required: true
+        type: string
+      tag:
+        description: 'Latest tag for upload'
+        required: true
+        type: string
+      label:
+        description: 'Release label for upload'
+        required: true
+        type: string
+
+jobs:
+
+  build-amd:
+    if: ${{inputs.type != 'arm' }}
+    runs-on: self-hosted
+    env:
+      LATEST_TAG: ${{inputs.tag}}
+      LABEL: ${{inputs.label}}
+      DOCKERHUB_ORG: tessellprod
+      REPO: "${{github.event.pull_request.base.repo.name}}"
+      GITHUB_USER: ${{ secrets.CIPIPELINE_NEXUS_USERNAME }}
+      GITHUB_TOKEN: ${{ secrets.CIPIPELINE_NEXUS_PASSWORD }}
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL  }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Login to docker
+        uses: docker/login-action@v2.1.0
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}   
+
+      - name: Build and Push
+        shell: bash
+        run: |
+          dockerBuild() {
+            set -e
+            IMAGE_NAME="$1"
+            FILE="$4"
+            if [[ "$FILE" == "null" ]]; then
+              FILE="./Dockerfile"
+            fi
+            echo "$FILE"
+            ./mvnw install -Dnative -DskipTests -Dquarkus.native.remote-container-build=true
+            docker build -f $FILE -t ${DOCKERHUB_ORG}/$IMAGE_NAME:${LATEST_TAG} .
+            docker push ${DOCKERHUB_ORG}/$IMAGE_NAME:${LATEST_TAG}
+            set +e
+          }
+          build() {
+            type="$1"
+            check=$(grep "$type" convoy.yaml)
+            if [[ -z "$check" ]]; then
+              return
+            fi
+            while IFS=$'\t' read -r name buildFunction ext version file _; do
+              echo "Name: $name"
+              echo "buildFunction: $buildFunction"
+              echo "Ext: $ext"
+              echo "Version: $version"
+              echo "dockerFile: $file"
+              $buildFunction "$name" "$ext" "$version" "$file"
+            done < <(yq e ".generates.$type[] | [.name, .buildFunction, .extension, .version, .dockerFile] | @tsv" convoy.yaml)
+          }
+          set +e
+          build "artifacts"
+          build "dockerImages"
+          build "helmCharts"
+
+      - name: Remove target
+        if: always()
+        run: |
+          set +e
+          sudo rm -rf target
+          docker container prune --force
+          docker volume prune --force
+
+      - name: Slack Notification
+        uses: act10ns/slack@v1.5.0
+        if: failure()
+        with:
+          status: ${{ job.status }}
+          steps: ${{ toJson(steps) }}
+          channel: ${{ secrets.SLACK_DEVOPS_CHANNEL }}
+
+  build-arm:
+    if: ${{inputs.type}} == 'arm'
+    runs-on: ARM64
+    env:
+      LATEST_TAG: ${{inputs.tag}}
+      LABEL: ${{inputs.label}}
+      NEXUS_SERVER: nexus.tessell.cloud
```

Comment:
Variable `NEXUS_SERVER_ENDPOINT` defined with value `nexus.tessell.cloud`; You can use the same at all places.